src/drop-down-menu.jsx

```diff
@@ -36,7 +37,8 @@ var DropDownMenu = React.createClass({
     return {
       open: false,
       isHovered: false,
-      selectedIndex: this.props.selectedIndex || 0
+      selectedIndex: this.props.selectedIndex || 0,
+      selectedItems: []
     }
```

Comment:
@xmityaz Sorry, My comment is on the wrong line. My point was for the getDefaultProps method.